chsdi/views/downloadkml.py

```diff
@@ -47,9 +46,6 @@ def __init__(self, request):
     @view_config(route_name='downloadkml', renderer='jsonp')
     def downloadkml(self):
 
-        if self.request.method == 'OPTIONS':
-            return Response(status=200)
-
```

Comment:
because of a misunderstanding on how pyramid works.

Reply:
mapgeoadmin validates CORS on this service.
It's a POST request that expects a body. But we should  have used a more generic approach for these since the beginning.